apps/admin/app/(auth)/forgot-password/_components/form.tsx

```diff
@@ -0,0 +1,67 @@
+"use client";
+
+import { AuthPage } from "@repo/ui/components";
+import { formSchema, TFormSchema } from "./form-schema";
+import { useForm } from "react-hook-form";
+import { ReactForm } from "@repo/ui/forms";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { Button, Input, Linktag, Textbox } from "@repo/ui/partials";
+import Link from "next/link";
+
+export function ForgotPasswordForm() {
+  const form = useForm<TFormSchema>({
+    resolver: zodResolver(formSchema),
+    defaultValues: {
+      email: "",
+    },
+  });
+
+  const onSubmit = () => {
```

Comment:
```suggestion
  const onSubmit = () => {
```
please remove comsole

Reply:
@yaju09 This is just to test it. This will definitely be removed while implementing the functionality

Reply:
Add a todo. @sidarth-23